src/parser.rs

```diff
@@ -93,54 +137,57 @@ fn parse_hex(input: &str) -> IResult<&str, Color> {
 }
 
 fn parse_numeric_rgb(input: &str) -> IResult<&str, Color> {
-    let (input, prefixed) = opt(tag("rgb("))(input)?;
+    let (input, prefixed) = opt(alt((tag("rgb("), tag("rgba("))))(input)?;
```

Comment:
Hm. So do we allow `rgba(1,2,3)` or `rgb(1,2,3,30%)` as well?

Reply:
Here's the spec according to the most recent [CSS Color Module draft](https://www.w3.org/TR/css-color-4/) from W3C:

```
rgb() = rgb( [<percentage> | none]{3} [ / [<alpha-value> | none] ]? ) |
  rgb( [<number> | none]{3} [ / [<alpha-value> | none] ]? )
<alpha-value> = <number> | <percentage>
```

> For legacy reasons, [rgb()](https://www.w3.org/TR/css-color-4/#funcdef-rgb) also supports a [legacy color syntax](https://www.w3.org/TR/css-color-4/#legacy-color-syntax) that separates all of its arguments with commas

> Also, an rgba() [legacy color syntax](https://www.w3.org/TR/css-color-4/#legacy-color-syntax) also exists, with an identical grammar and behavior to [rgb()](https://www.w3.org/TR/css-color-4/#funcdef-rgb).

The `a`-suffixed variants of `rgb` and `hsl` have been deprecated in that Level 4 draft, but to maintain compatibility they essentially function identically to their non-suffixed counterparts. While `rgba()` with 3 values and `rgb()` with 4 values look a bit odd, they're accepted by most modern browsers (checked Chrome and Firefox) so I'd lean towards allowing them.